README.md

```diff
@@ -1 +1,186 @@
-# react-app-preloader
\ No newline at end of file
+# react-wait-for-react
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/react-wait-for-react
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-wait-for-react.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-wait-for-react.svg
+[build-status-url]:https://github.com/moxystudio/next-with-moxy/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/next-with-moxy/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-wait-for-react
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-wait-for-react/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-wait-for-react
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-wait-for-react.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-wait-for-react?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-wait-for-react.svg
+
+Easily render a loader while your React app is loading, optionally waiting for a promise as well.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-wait-for-react
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+Certain apps or pages have impactful experiences. These experiences can make the total bundle size larger as they pack possibly large dependencies and media assets, such as 3D objects and audio files.
+
+It's then often normal to preload all the required files for an uninterrupted experience. `@moxy/wait-for-react` is a library that makes it easy to display a loader before your static or server-side rendered app becomes interactive, and optionally until all the required files are loaded (via a promise).
```

Comment:
`@moxy/wait-for-react` is a...

Should it be: `@moxy/react-wait-for-react`?

Reply:
Nice catch, fixed.